src/plugins/janus_sip.c

```diff
@@ -4947,6 +4958,16 @@ static void *janus_sip_handler(void *data) {
 			/* Notify the operation */
 			result = json_object();
 			json_object_set_new(result, "event", json_string("infosent"));
+		} else if(!strcasecmp(request_text, "send_ringing")) {
+			if(session->status != janus_sip_call_status_invited
+				&& session->status != janus_sip_call_status_progress) {
```

Comment:
Code style nit: the `&&` must be on the previous line, and this second line must have a second tab of indentation.